src/internal/connector/exchange/data_collections_test.go

```diff
@@ -604,6 +609,7 @@ func (suite *DataCollectionsIntegrationSuite) TestEventsSerializationRegression(
 				test.scope,
 				DeltaPaths{},
 				control.Defaults(),
+				true,
```

Comment:
I think we need some `false` case tests in regression to be fully comfortable with this change.

Reply:
Augmented existing tests to test both in e9b5de8c3